src/form/acr.jl

```diff
@@ -47,6 +121,31 @@ function constraint_mc_voltage_magnitude_bounds(pm::_PM.AbstractACRModel, n::Int
     end
 end
 
+"bus voltage on/off constraint for load shed problem"
+function constraint_mc_bus_voltage_on_off(pm::_PM.AbstractACRModel; nw::Int=pm.cnw, kwargs...)
+    for (i,bus) in ref(pm, nw, :bus)
+        constraint_mc_bus_voltage_magnitude_on_off(pm, i; nw=nw)
+    end
+end
+
+
+"on/off bus voltage magnitude constraint"
+function constraint_mc_bus_voltage_magnitude_on_off(pm::_PM.AbstractACRModel, n::Int, i::Int, vmin, vmax)
+    vr = var(pm, n, :vr, i)
+    vi = var(pm, n, :vi, i)
+    z_voltage = var(pm, n, :z_voltage, i)
+
+    for c in conductor_ids(pm, n)
+        if isfinite(vmax[c])
```

Comment:
This is used this way because by default we don't have voltage bounds anymore in the engineering model, which, because PowerModels always expects them, we convert to vmin = 0 and vmax=Inf for each phase